idseq_dag/steps/blast_contigs.py

```diff
@@ -68,10 +74,11 @@ def run(self):
         if self.additional_files.get("deuterostome_db"):
             deuterostome_db = s3.fetch_from_s3(self.additional_files["deuterostome_db"],
                                                self.ref_dir_local, allow_s3mi=False) # Too small for s3mi
-        with log.log_context("PipelineStepBlastContigs", {"substep": "generate_taxon_count_json_from_m8", "db_type": db_type, "refined_counts": refined_counts}):
-            m8.generate_taxon_count_json_from_m8(refined_m8, refined_hit_summary,
-                                                evalue_type, db_type.upper(),
-                                                lineage_db, deuterostome_db, refined_counts)
+        with TraceLock("PipelineStepBlastContigs-CYA", PipelineStepBlastContigs.cya_lock, debug=false):
```

Comment:
@davidrissato is this how to properly use tracelock?

Reply:
Yes, this is the correct way